app/api/v1/conversations.py

```
@@ -8,20 +8,20 @@
 
 from app.db.config.database import get_async_session
 from app.db.models import Conversation
+from app.services.chat_service import chat_service
```

Ruff (0.12.2)

`app.services.chat_service.chat_service` imported but unused

Remove unused import: `app.services.chat_service.chat_service`

(F401)